src/runtime_src/tools/scripts/xrtdeps-win22.py

```diff
@@ -724,6 +734,63 @@ def getBuildAndInstallLibrary(self, verbose):
   def skipBuild(self):
     return self.skipBuildInstall == False
 
+#==============================================================================
+# -- Class: pybind11_package  -------------------------------------------------
+#==============================================================================
+class pybind11_package():
+    """
+    This class contains methods to check for a pybind11 installation and to
+    install the package using pip. Note that even if a pybind11 version exists,
+    if it doesn't match the expected_version parameter it will be overwritten.
+
+    Parameters
+    ----------
+    expected_version : str
+        Desired pybind11 version
+    """
+
+    def __init__(self, expected_version='2.10.4'):
+        self.installed = False
+        self.expected_version = expected_version
+        
+        if self.check_installed():
+            print(f"Successfully found pybind11 installation")
+            pass
+        else: 
+          print(f"Did not find a compatible installation of pybind11")
+          print(f"Trying to install pybind11=={self.expected_version} from pypi...")
```

Comment:
The message reminds me of the phrase "Do or do not.  There is no try" - Yoda.

It might be better to paraphrase the message as follows:
```suggestion
          print(f"Installing pybind11 ({self.expected_version}) via pypi...")
```